sal_interfaces/MTM2/MTM2_Commands.xml

```diff
@@ -101,4 +101,46 @@
     </item>
 </SALCommand>
 
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_switchForceBalanceSystem</EFDB_Topic>
+    <Alias>switchForceBalanceSystem</Alias>
+    <Description>Switch the force balance (FB) system.</Description>
+    <item>
+        <EFDB_Name>status</EFDB_Name>
+        <Description>0: turn off FB. 1: turn on FB.</Description>
+        <IDL_Type>int</IDL_Type>
```

Comment:
If it is on/off, I would recommend using a `boolean`.

Reply:
Done in commit: 9c5cc3f. Thanks!